submission/persistence/src/main/kotlin/ac/uk/ebi/biostd/persistence/service/filesystem/RefFilesService.kt

```diff
@@ -18,44 +18,45 @@ import ebi.ac.uk.io.FileUtils.reCreateDirectory
 import ebi.ac.uk.paths.FILES_PATH
 import ebi.ac.uk.paths.SubmissionFolderResolver
 import java.io.File
+import java.nio.file.Path
 
-class FilePersistenceService(
+class RefFilesService(
```

Comment:
I think the previous name was clearer. Also, `RefFile` implies a direct correlation with the database table with the same name which is only used for regular files, not file list referenced ones